### PR TITLE
doc: update README with recent changes

### PR DESCRIPTION
**Pull request by mrts**
Refs WE2-274

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,10 +171,26 @@ how to use input-output mode, it can be run with:
...
+      echo 'logging=true' > ~/.config/RIA/web-eid.conf
+
+- in macOS, run the following command in the console:
+
+      defaults write eu.web-eid.web-eid logging true
+      # FIXME: is this correct? How to enable Safari extension logging?
+
+- in Windows, add the following registry key:
+
+      [HKEY_LOCAL_MACHINE\SOFTWARE\RIA\web-eid]
```
Comment (metsma) [inline README.md:187]: HKEY_CURRENT_USER

File: README.md [hunk 2]
```diff
@@ -147,10 +171,26 @@ how to use input-output mode, it can be run with:
...
+      "logging"="true"
+
 The application writes logs to
 
 - `~/.local/share/RIA/web-eid/web-eid.log` in Linux
-- `~/Library/Application Support/RIA/web-eid/web-eid.log` in macOS
+- `~/Library/Application Support/RIA/web-eid/web-eid.log` in macOS # FIXME: is this correct?
```
Comment (metsma) [inline README.md:193]: For unsigned chrome native host: Yes

**Review by metsma — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mrts) [reply README.md:187]: Thanks, fixed!

File: README.md (hunk 2 above)
Comment (mrts) [reply README.md]: Thanks for confirming!

**Review by metsma — APPROVED** [review]

**Pull request merged**